frontend/components/CheckBoxes/style.js

```diff
@@ -0,0 +1,21 @@
+import { css } from 'glamor';
+import { themeConfig } from '@shopgate/pwa-common/helpers/config';
+
+const { colors } = themeConfig;
+
+const wrapper = css({
+  background: colors.shade8,
+}).toString();
+
+const content = css({
+  paddingTop: '15px',
+  width: '80%',
```

Comment:
I'd suggest having 100% and variables.gap.big padding instead

Reply:
updated